### DISCUSSION TITLE
error when i create p12 certificate using adt  certificate decoding error

### DISCUSSION THREAD

**Discussion by moazatiani** [airsdk/Adobe-Runtime-Support#1559] [Q&A]
hi every one 
today  when i testing to create certificate file p12 using adt    i follow all steps  
below my code 
adt -certificate -cn salem -ou samer -o dajer -c US -validityPeriod 20 2048-RSA moaz.p12 micro9921
but  still  error like this : 
certificate decoding error

**Comment by ajwfrost** [comment]
Hi
What are you doing when you get the "certificate decoding error"? I'm assuming the "adt -certificate" command works okay, and generates you the moaz.p12 file?
thanks
